pd2jira_function/pd2jira/escalate_to_pd.py

```diff
@@ -0,0 +1,90 @@
+import os
+import logging
+import urllib
+import json
+import boto3
+import datetime
+import pdpyras
+from jira import JIRA
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', logging.DEBUG))
+
+for handler in logger.handlers:
```

Comment:
does this do anything for you? I've never been able to change the log format in a lambda